operators/multiclusterobservability/manifests/base/alertmanager/alert_rules.yaml

```
@@ -18,7 +18,7 @@ data:
             cluster: "{{ $labels.cluster }}"
             clusterID: "{{ $labels.clusterID }}"
             PersistentVolumeClaim: "{{ $labels.persistentvolumeclaim }}"
-            severity: critical
+            severity: informational
```

Alert severity level for `informational` is `info`:
```suggestion
            severity: info
```

---

done